runtimes/eoapi/stac/eoapi/stac/client.py

```diff
@@ -287,67 +285,9 @@ async def landing_page(
             API landing page, serving as an entry point to the API.
 
         """
-        base_url = get_base_url(request)
-
-        landing_page = self._landing_page(
-            base_url=base_url,
-            conformance_classes=self.conformance_classes(),
-            extension_schemas=[],
-        )
-
-        # Add Queryables link
-        if self.extension_is_enabled("FilterExtension") or self.extension_is_enabled(
-            "SearchFilterExtension"
-        ):
-            landing_page["links"].append(
-                {
-                    "rel": Relations.queryables.value,
-                    "type": MimeTypes.jsonschema.value,
-                    "title": "Queryables",
-                    "href": urljoin(base_url, "queryables"),
-                }
-            )
-
-        # Add Aggregation links
-        if self.extension_is_enabled("AggregationExtension"):
-            landing_page["links"].extend(
-                [
-                    {
-                        "rel": "aggregate",
-                        "type": "application/json",
-                        "title": "Aggregate",
-                        "href": urljoin(base_url, "aggregate"),
-                    },
-                    {
-                        "rel": "aggregations",
-                        "type": "application/json",
-                        "title": "Aggregations",
-                        "href": urljoin(base_url, "aggregations"),
-                    },
-                ]
-            )
-
-        # Add OpenAPI URL
-        landing_page["links"].append(
-            {
-                "rel": Relations.service_desc.value,
-                "type": MimeTypes.openapi.value,
-                "title": "OpenAPI service description",
-                "href": str(request.url_for("openapi")),
-            }
-        )
+        request: Request = kwargs["request"]
```

Comment:
now that `collections` has been removed we don't need a custom `landing_page` and can call `super().landing_page(**kwargs)`

Reply:
got it - but we still need this custom method because we have the html response type